excel2sbol/excel2sbol/compiler_test.py

```diff
@@ -197,7 +197,7 @@ def parse_objects3(col_read_df, to_convert, compiled_sheets,
             raise KeyError(f'The sheet "{sht}" has no column with sbol_objectType as type. Thus the following error was raised: {e}')
 
         try:
-            mol_type_col = sht_df.loc[col_read_df['SBOL Term'] == 'sbol_type']['Column Name'].values[0]
+            mol_type_col = sht_df.loc[col_read_df['SBOL Term'] == 'sbol_types']['Column Name'].values[0]
```

Comment:
what does this change do?

Reply:
If sbol_type is kept, either in this place or on the column definitions sheet, it isn't recognized. When changed to sbol_types, it's recognized and put under the correct association, so I believe the plural is the correct term just from experimentation.